test/e2e/positive.go

```diff
@@ -293,14 +397,15 @@ func getNodeResourceTopology(tc *topologyclientset.Clientset, namespace, name st
 	return nrt
 }
 
-func deploy() error {
+func deploy(updaterArg string) error {
```

Comment:
better to pass through the flag and build the right commandline internally instead of adding random stuff to the cmdline -even if this is a very controlled environment

Reply:
You mean passing the whole command line?

Reply:
better to pass an enum-value here (like `RTE` or `NFD`) and let the function build and add the proper option to the command line